spec/test_helper.rb

```diff
@@ -0,0 +1,21 @@
+class TestHelper
+  def self.get_first_row_event_from_client(ecco_client)
```

Comment:
> I think that will lead to confusion regarding what is returned from the method. Otherwise I like the proposed change.

`start_client_get_first_row_event_then_stop_client_and_then_return_the_event` 

Much better :P